dsl/scripts/jobs.groovy

```diff
@@ -194,11 +194,12 @@ def vegavisorRestartNetworkParams(args=[:]) {
 
 def vegavisorRestartNodeParams(args=[:]) {
     def choices = [
-        'restart-node': 'regular restart',
+        'restart-node': 'regular restart from local snapshot',
```

Comment:
`local snapshot` is not always true.
When we select `UNSAFE_RESET_ALL`, then it is going to delete everything locally, including snapshots, and will instead start from block 0.